index.html

```diff
@@ -319,47 +319,6 @@ <h2 class="title_contact">Contact me</h2>
 			</div>
```

Comment:
- [ ] Please make sure the email input has a required attribute. I would also suggest removing the maxlength from the email input as this might prevent longer emails from being used.